data/collector.go

```diff
@@ -34,6 +34,7 @@ var sleepCaptureCutoff = 2 * time.Millisecond
 type CaptureFunc func(ctx context.Context, params map[string]*anypb.Any) (interface{}, error)
 
 // FromDMContextKey is used to check whether the context is from data management.
+// Deprecated: use a camera.Extra with camera.NewContext instead.
```

Comment:
Previously left a comment that we could skip the change I suggested, but I actually think this would be necessary to stage together - otherwise we’ll have to update the modules twice — once to grab from extra once to grab with the new key. 

This is also a huge usability improvement so that users aren't using different keys in different places for the same purpose.

Reply:
Per our discussion, pushing this as is now in time for our release.